hyperspy/roi.py

```diff
@@ -688,6 +703,24 @@ def _set_default_values(self, signal, axes=None):
 
     @property
     def parameters(self):
+        """dict : ROI parameters
+        Dictionary containing the parameters that define the current ROI.
+
+        For a ``Point1DROI``, the dictionary contains:
+        - ``position`` : float
+        The coordinate of the point along the axis.
+
+        Notes
+        -----
+        This property is intended for serialization and restoration of ROIs.
+
+        Examples
+        --------
+        >>> from hyperspy.roi import Point1DROI
+        >>> roi = Point1DROI(position=15)
```

Comment:
Running this example returns:
```
Point1DROI.__init__() got an unexpected keyword argument 'position'
```

I guess the parameter is actually called `value`. Please check the examples once in a python console.